marimo/_server/notebook/serializer.py

```diff
@@ -62,7 +62,7 @@ class MarkdownNotebookSerializer(NotebookSerializer):
 
     def serialize(self, notebook: NotebookSerializationV1) -> str:
         """Serialize notebook to Markdown format."""
-        return convert_from_ir_to_markdown(notebook)
+        return convert_from_ir_to_markdown(notebook, notebook.filename)
```

Comment:
this can be reverted right?

Reply:
Yes, the regex is loose enough to get both cases without passing in filename information